src/sentry/api/endpoints/project_index.py

```diff
@@ -55,6 +55,10 @@ def get(self, request: Request) -> Response:
             else:
                 queryset = queryset.filter(teams__organizationmember__user_id=request.user.id)
 
+            org_id_filter = request.GET.get("organization_id", None)
```

Comment:
```suggestion
            org_id_filter = request.GET.get("organizationId", None)
```

Most API/query parameters are camelBacked, we should follow that convention as well.